<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QMainWindow" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Student Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Firstname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Value_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Value_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Lastname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Value_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Value_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>291</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);
 border-radius: 8px
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_Value_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>440</y>
       <width>71</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>71</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Value_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Create An Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_4</zorder>
   <zorder>label_8</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_Value_4</zorder>
   <zorder>lineEdit_Value_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit_Value_3</zorder>
   <zorder>lineEdit_Value_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
